src/Simulation/TargetDefinitions/Decompositions/ApplyUncontrolledX.qs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+    open Microsoft.Quantum.Math;
+
+    /// # Summary
+    /// Applies the Pauli $X$ gate. Note that the Controlled functor is not supported.
+    ///
+    /// # Description
+    /// \begin{align}
+    ///     \sigma_x \mathrel{:=}
+    ///     \begin{bmatrix}
+    ///         0 & 1 \\\\
+    ///         1 & 0
+    ///     \end{bmatrix}.
+    /// \end{align}
+    ///
+    /// # Input
+    /// ## qubit
+    /// Qubit to which the gate should be applied.
+    internal operation ApplyUncontrolledX (qubit : Qubit) : Unit is Adj {
+        body (...) {
+            ApplyUncontrolledRx(PI(), qubit);
+        }
+        adjoint self;
+    }
+}
```

Comment:
(nit)
A thought crawls into my mind: May be in the future, instead of using `Uncontrolled` and `Controlled` it makes sense to use \<Nothing> and `Controlled`? E.g. instead of using 
`ApplyUncontrolledX` and `ApplyControlledX` to use 
`ApplyX` and `ApplyControlledX`.

Reply:
Yeah, that's a good point, and I've been debating doing a broader rename of the files too. My initial thinking was to use "Uncontrolled" to make explicit the fact that the operation did not support the Controlled functor, but now it just seems unnecessary and wordy. I think a broader rename could be a follow up PR.